controllers/__init__.py

```diff
@@ -182,3 +183,11 @@ def upsert_flow_check(self, dpid: str, state="active") -> Optional[dict]:
     def get_flow_check(self, dpid: str, state="active") -> Optional[dict]:
         """Get flow check."""
         return self.db.flow_checks.find_one({"_id": dpid, "state": state})
+
+    def find_flows(
```

Comment:
@viniarck, You are right, this structure is better because of the way this endpoint will be used.

Here an example from Postman:

`...v2/stored_flows?dpid=00:00:00:00:00:00:00:03&state=installed`

```
{
    "00:00:00:00:00:00:00:03": [
        {
            "flow": {
                "actions": [
                    {
                        "action_type": "push_vlan",
                        "tag_type": "s"
                    },
                    {
                        "action_type": "set_vlan",
                        "vlan_id": 4001
                    },
                    {
                        "action_type": "output",
                        "port": 2
                    }
                ],
                "cookie": 12310228866111668291,
                "hard_timeout": 0,
                "idle_timeout": 0,
                "match": {
                    "in_port": 1
                },
                "priority": 32768,
                "table_id": 0
            },
            "flow_id": "eb1003186fedf38a0cfe541a332b2014",
            "id": "f0224f84f3651423176b53cf83ba8e5c",
            "inserted_at": "2022-09-13T17:48:51",
            "state": "installed",
            "switch": "00:00:00:00:00:00:00:03",
            "updated_at": "2022-10-04T20:47:14"
        },
        {
            "flow": {
                "actions": [
                    {
                        "action_type": "pop_vlan"
                    },
                    {
                        "action_type": "output",
                        "port": 1
                    }
                ],
                "cookie": 12310228866111668291,
                "hard_timeout": 0,
                "idle_timeout": 0,
                "match": {
                    "dl_vlan": 4001,
                    "in_port": 2
                },
                "priority": 32768,
                "table_id": 0
            },
            "flow_id": "b6f7b4c6e0a3817956c75eeec36c88a3",
            "id": "d477eb38170a452c98f76ef720de002e",
            "inserted_at": "2022-09-22T21:00:44",
            "state": "installed",
            "switch": "00:00:00:00:00:00:00:03",
            "updated_at": "2022-10-04T20:14:18"
        },
        {
            "flow": {
                "actions": [
                    {
                        "action_type": "pop_vlan"
                    },
                    {
                        "action_type": "output",
                        "port": 1
                    }
                ],
                "cookie": 12313817929802993474,
                "hard_timeout": 0,
                "idle_timeout": 0,
                "match": {
                    "dl_vlan": 2881,
                    "in_port": 2
                },
                "priority": 32768,
                "table_id": 0
            },
            "flow_id": "4194481ecd9d3ea1db03368ddf8f54d8",
            "id": "6a579c78188f09c6a7c266427f9e8f30",
            "inserted_at": "2022-10-04T20:14:12",
            "state": "installed",
            "switch": "00:00:00:00:00:00:00:03",
            "updated_at": "2022-10-04T20:14:12"
        },
        {
            "flow": {
                "actions": [
                    {
                        "action_type": "push_vlan",
                        "tag_type": "s"
                    },
                    {
                        "action_type": "set_vlan",
                        "vlan_id": 2881
                    },
                    {
                        "action_type": "output",
                        "port": 2
                    }
                ],
                "cookie": 12313817929802993474,
                "hard_timeout": 0,
                "idle_timeout": 0,
                "match": {
                    "in_port": 1
                },
                "priority": 32768,
                "table_id": 0
            },
            "flow_id": "f53a489b8c5bffcd565fa8f277893825",
            "id": "2fe1c0a67c67215617cf0a4964bb1dc0",
            "inserted_at": "2022-10-04T20:14:12",
            "state": "installed",
            "switch": "00:00:00:00:00:00:00:03",
            "updated_at": "2022-10-04T20:49:14"
        },
        {
            "flow": {
                "actions": [
                    {
                        "action_type": "output",
                        "port": 4294967293
                    }
                ],
                "cookie": 12321848580485677059,
                "hard_timeout": 0,
                "idle_timeout": 0,
                "match": {
                    "dl_type": 35020,
                    "dl_vlan": 3799
                },
                "priority": 1000,
                "table_id": 0
            },
            "flow_id": "48083b8562161a2923d5edda794999f4",
            "id": "f1fff6c7ba4d2ac9bc1e2dfe6c91d8af",
            "inserted_at": "2022-09-09T18:20:27",
            "state": "installed",
            "switch": "00:00:00:00:00:00:00:03",
            "updated_at": "2022-10-04T20:14:11"
        }
    ]
}
```